.github/workflows/run-tests.yml

```diff
@@ -1,10 +1,26 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
-
 name: Run Tests
 
 on: [push, pull_request]
 
 jobs:
   call_run_tests:
     uses: yext/slapshot-reusable-workflows/.github/workflows/run_tests.yml@v1
+    with:
+      node_matrix: '["16.x"]'
+
+  # It's necessary to use v12 of the react-testing-library since v13 was updated to only support react 18
+  call_run_tests-react-17:
```

Comment:
I made an item for running tests in React v17, potentially looking at `legacyRoot` for rendering in pre18 versions. but since we need to change the react version install, directly `@testing-library/react@12` make sense. Nice!